packages/react-router/__tests__/Routes-test.tsx

```diff
@@ -16,7 +16,12 @@ describe("A <Routes>", () => {
       </Router>
     );
 
-    expect(renderer.toJSON()).toMatchSnapshot();
+    let json = renderer.toJSON();
+    if (!json || Array.isArray(json)) {
+      throw Error("Unexpected JSON from `createTestRenderer`");
+    }
+    expect(json).toMatchSnapshot();
+    expect(json.children?.includes("Home")).toBe(true);
```

Comment:
Unrelated to animation stuff, but I figure this makes the tests a little stronger so I added these assertions before copying this as a base for the animation tests.